Formula/p11-kit.rb

```diff
@@ -0,0 +1,27 @@
+class P11Kit < Formula
+  desc "Library to load and enumerate PKCS#11 modules"
+  homepage "https://p11-glue.freedesktop.org"
+  url "https://p11-glue.freedesktop.org/releases/p11-kit-0.23.2.tar.gz"
+  sha256 "ba726ea8303c97467a33fca50ee79b7b35212964be808ecf9b145e9042fdfaf0"
+
+  depends_on "pkg-config" => :build
+  depends_on "libffi"
```

Comment:
libffi and libtasn1 can both be `:recommended`, if we care about providing people with that flexibility.